proposals/98:raise-psm-mintlimit/test.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source /usr/src/upgrade-test-scripts/env_setup.sh
+
+pair="IST.USDC"
+NOBLE_USDC_DENOM="ibc/FE98AAD68F02F03565E9FA39A5E627946699B2B07115889ED812D8BA639576A9"
```

Shellcheck (0.11.0)

[warning] NOBLE_USDC_DENOM appears unused. Verify use (or export if used externally).

(SC2034)